test/Serilog.Sinks.RollingFile.Tests/project.json

```diff
@@ -24,6 +23,7 @@
         "dnxcore50",
         "portable-net45+win8"
       ]
-    }
+    },
+    "net4.5.2": {}
```

Comment:
Moving the .NET Core App framework to the top was needed to get the tests to run, as the test discovery failed with the following error:

> Unable to start C:\Program Files\dotnet\dotnet.exe
dotnet-test Error: 0 : [ReportingChannel]: Waiting for message failed System.IO.IOException: Unable to read data from the transport connection: An established connection was aborted by the software in your host machine. ---> System.Net.Sockets.SocketException: An established connection was aborted by the software in your host machine
   at System.Net.Sockets.Socket.Receive(Byte[] buffer, Int32 offset, Int32 size, SocketFlags socketFlags)
   at System.Net.Sockets.NetworkStream.Read(Byte[] buffer, Int32 offset, Int32 size)
   --- End of inner exception stack trace ---
   at System.Net.Sockets.NetworkStream.Read(Byte[] buffer, Int32 offset, Int32 size)
   at System.IO.Stream.ReadByte()
   at System.IO.BinaryReader.ReadByte()
   at System.IO.BinaryReader.Read7BitEncodedInt()
   at System.IO.BinaryReader.ReadString()
   at Microsoft.DotNet.Tools.Test.ReportingChannel.ReadMessages()

This looks related to https://github.com/xunit/xunit/issues/900, which is more than likely a bug with the .NET Core tooling somewhere and not with xUnit.net itself. For now, this (silly) workaround seems to do the trick.